qcodes/dataset/sqlite/connection.py

```diff
@@ -27,8 +27,10 @@ class ConnectionPlus(wrapt.ObjectProxy):
         atomic_in_progress: a bool describing whether the connection is
             currently in the middle of an atomic block of transactions, thus
             allowing to nest `atomic` context managers
+        path_to_dbfile: Path to the database file of the connection.
     """
     atomic_in_progress: bool = False
+    path_to_dbfile = ''
```

Comment:
could you add some assertions for this to [ConnectionPlus tests as well](https://github.com/QCoDeS/Qcodes/blob/9bf5110c006cb5b01e5368405fe952fe25a613aa/qcodes/tests/dataset/test_sqlite_connection.py#L42) ?